include/tracer.hrl

```diff
@@ -0,0 +1,47 @@
+%% macros for tracing
+%% register a tracer for an application with opentelemetry:register_application_tracer(AppName)
+
+-define(current_tracer, opentelemetry:get_tracer(?MODULE)).
+-define(current_span_ctx, ot_tracer:current_span_ctx(?current_tracer)).
+
+-define(start_span(SpanName),
```

Comment:
Ok, personally I really hate that they do that for logging. All caps for the rare constant makes sense and isn't disruptive.

I don't like yelling when I log or start/modify a span :).

Like in common test it is `?config(key, Config)` and not `?CONFIG(key, Config)`.

But since logging is all caps I'll probably accept tradition and change these to all caps... Unless I can get some backup on the use of lowercase from someone -- I doubt I can.

Reply:
Hm, weird, old docs use lowercase, maybe that is why CT uses `?config`, it is legacy.

Reply:
I'd just say to export both of them at once since macros are case-sensitive.